packages/react/src/drawer/docs/overview.mdx

```diff
@@ -72,6 +72,46 @@ The Drawer can be closed by either:
 
 When closing the Drawer, focus should be returned to the button that opened the Drawer.
 
+## Drawer with muted overlay
+
+The Drawer overlay defaults to 80% opacity however by adding `mutedOverlay={true}` the opacity can be reduced to 30%. This makes it easier for sighted users to see the content underneath the Drawer overlay so that context can more easily be retained with the main page content.
+
+This is available for when sighted users with cognative disabilites need to retain context with the main page content while interacting with the Drawer content.
+
+Only apply this when additional context with the main page content is required.
```

Comment:
I think we should have this in the Dos and Don'ts. Something like:

Do: Use the muted overlay when making it easier for users to retain the context of the underlying page is beneficial, especially for sighted users with cognitive disabilities.

Don't: Use the muted overlay because you prefer the lighter appearance.

@adrian-yee @danger17  Thoughts? Care to wordsmith?

Reply:
I've added the following to the do and don't lists

DO

- use the muted overlay when it is beneficial to retain context of the underlying page content, especially for sighted users with cognitive disabilities

DON'T

- use the muted overlay because you prefer the lighter appearance

Reply:
DO

- use the muted overlay when it is beneficial to retain the user's context of the underlying page content, especially for sighted users with cognitive disabilities